### PR TITLE
Add SIG Architecture Conformance spotlight interview

### PR DESCRIPTION
**Pull request by fsmunoz**
Spotlight article on SIG Architecture: Conformance, the first of the series.

Contributor-site PR: https://github.com/kubernetes/contributor-site/pull/431 ; content is currently in sync, and I will update this one with the result of the review in the `contributor-site` PR, so for general changes please add to that one (changes specific to the `website`  are welcomed here though).

### CODE REVIEW

File: content/en/blog/_posts/2023-09-28-sig-architecture-conformance-spotlight.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+---
+layout: blog
+title: "Spotlight on SIG Architecture: Conformance"
+slug: sig-architecture-conformance-spotlight-2023
+date: 2023-09-28
+author: "Frederico Muñoz (SAS Institute)"
```
Comment (windsonsea) [inline content/en/blog/_posts/2023-09-28-sig-architecture-conformance-spotlight.md:6]: Line 6 can be removed, which is duplicated with the first body text.

File: content/en/blog/_posts/2023-09-28-sig-architecture-conformance-spotlight.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+_This is the first interview of a SIG Architecture Spotlight series
+that will cover the different subprojects. We start with the SIG
+Architecture: Conformance subproject_
+
+In this [SIG
+Architecture](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md)
+spotlight, we talked with [Riaan
+Kleinhans](https://github.com/Riaankl) (ii-Team), Lead for the
+[Conformance
+sub-project](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md#conformance-definition-1).
```
Comment (windsonsea) [inline content/en/blog/_posts/2023-09-28-sig-architecture-conformance-spotlight.md:21]:
```suggestion
In this [SIG Architecture](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md)
spotlight, we talked with [Riaan Kleinhans](https://github.com/Riaankl) (ii-Team), Lead for the
[Conformance sub-project](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md#conformance-definition-1).
```
Do not break the link text though it has the same effect.

File: content/en/blog/_posts/2023-09-28-sig-architecture-conformance-spotlight.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+standardised tests and promoting consistent behaviour and
+functionality, the Conformance subproject ensures a reliable and
+compatible Kubernetes ecosystem for developers and users alike.
+
+## More on the Conformance Test Suite
+
+**FSM**: A part of providing those standardised tests is, I believe,
+the [Conformance Test
+Suite](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md). Could
+you explain what it is and its importance?
```
Comment (windsonsea) [inline content/en/blog/_posts/2023-09-28-sig-architecture-conformance-spotlight.md:72]:
```suggestion
the [Conformance Test Suite](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md).
Could you explain what it is and its importance?
```

File: content/en/blog/_posts/2023-09-28-sig-architecture-conformance-spotlight.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+bit about yourself, your role and how you got involved in Kubernetes.
+
+**Riaan Kleinhans (RK)**: Hi! My name is Riaan Kleinhans and I live in
+South Africa. I am the Project manager for the [ii-Team](ii.nz) in New
+Zealand. When I joined ii the plan was to move to New Zealand in April
+2020 and then Covid happened. Fortunately, being a flexible and
+dynamic team we were able to make it work remotely and in very
+different time zones.
+
+The ii team have been tasked with managing the Kubernetes Conformance
```
Comment (AnshRoshan) [inline content/en/blog/_posts/2023-09-28-sig-architecture-conformance-spotlight.md:35]:
```suggestion
The ii team has been tasked with managing the Kubernetes Conformance
```

File: content/en/blog/_posts/2023-09-28-sig-architecture-conformance-spotlight.md
Comment (sftim) [inline content/en/blog/_posts/2023-09-28-sig-architecture-conformance-spotlight.md:1]: Hi. Can we push this back at least 7 days to find blog team review capacity?

I know that'll mean also pushing back the contributor site blog article that this mirrors.

**Comment by sftim** [comment]
/hold
publication date not yet confirmed

File: content/en/blog/_posts/2023-09-28-sig-architecture-conformance-spotlight.md (hunk 1 above)
Comment (sftim) [inline content/en/blog/_posts/2023-09-28-sig-architecture-conformance-spotlight.md:6]:
```suggestion
canonicalUrl: https://www.k8s.dev/blog/2023/10/05/sig-architecture-conformance-spotlight-2023/
```

**Comment by sftim** [comment]
_Previewed this locally; it looks OK to me_

/approve

**Pull request merged**